### PR TITLE
ViewReplies component

### LINKED ISSUE #83: View Reply Component (2)
Technical Background:
- As we are ready to work on the reply feature, we have to allow users to view replies on any comment

Acceptance Criteria:
- Create a component such that it displays replies under each comment

### PR DESCRIPTION
**Pull request by nchen000**
### Issues
closes #83 

### Overview of PR

- [x] add `ViewReplies` component
- [x] style `ViewReplies`
- [x] add documentation
- [x] add a new called `Comment`

## UI change
image

### Architecture Changes
- `ViewReplies` accepts prop, `replies`: an array of `reply` object
- `Comment` is extracted as a child component of `ViewComment`

### CODE REVIEW

File: src/components/Replies/ViewReplies/style.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  reply: {
+    width: 'calc(100% - 48px)',
+    marginLeft: theme.spacing(6),
+    background: '#f6f7fa',
+  },
```
Comment (uting98) [inline src/components/Replies/ViewReplies/style.js]: add `border: '1px solid',  borderRadius: 5,` to the reply helps to add the border styling

File: src/components/Replies/ViewReplies/ViewReplies.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          primary={(
+            <>
+              <Typography
+                component="span"
+                className={classes.left}
+                color="textPrimary"
+              >
+                {reply.username}
+                {reply.createdAt}
+                {reply.username}
```
Comment (uting98) [inline src/components/Replies/ViewReplies/ViewReplies.js]: Duplicate, should be remove

**Comment by nchen000** [comment]
> There is a bug, plz figure it out
> image

Please make sure the dummy data generated by backend follow the same format as before. This bug only seems to appear on `/viewpost/196`

File: src/components/Replies/ViewReplies/ViewReplies.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+} from 'prop-types';
+import useStyles from './style';
+
+export const ViewReplies = ({ replies }) => {
+  const classes = useStyles();
+  const avatarURL = 'http://api.adorable.io/avatar/50/';
+
+  return (
+    replies && replies.map((reply) => (
+      // eslint-disable-next-line react/no-array-index-key
```
Comment (uting98) [inline src/components/Replies/ViewReplies/ViewReplies.js]: unnecessary eslint disable

**Pull request merged**